notebooks/index.ipynb

```diff
@@ -65,7 +65,7 @@
     "\n",
```

Comment:
<p>For some reason, when I test downloading the notebook it actually downloads a .txt file instead of a .ipynb one. Did you observe the same?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>it worked for me but I can add a comment "make sure to save it with the ending *.ipynb?</p>

Reply:
<p>yeah adding the comment would be good</p>